NewForce-Cohort-8/react-nutshell-oracles-of-the-living
# Dd chat1

Initial chat module. To test:

1. Make sure that you can access "Messages" via Navbar
2. Type a message in the "Add Message" input and click the "Add Message" button
3. Ensure that your message is posted underneath "📝Message Board📝"
4. Ensure that your message has a delete button
5. Before testing delete button, log out and log back in as a different user
6. Ensure that the first user's message is visible, but without a delete button
7. Post a new message as the second user
8. Ensure that the new message (from the second user) populates underneath the first user's message
9. At this point, because you should still be logged in as the second user, you should see a delete button for the second user's message
10. Click the delete button to delete the second user's message
11. Ensure that only the first user's message remains

For optional additional testing: 
you can sign in as additional users and leave additional messages. Initial (author) testing completed using 5 users. The following criteria should always be true:

1. Oldest messages appear at the TOP of the page and newer messages appear beneath the previous message (user agnositc)
2. Messages from the user who is currently logged in (and only the current user) should present with a delete button
3. When there are too many messages to fit on a single screen, all messages still present on page but can be navigated via scrollbar

****NOTE**: per PM Sarah, reloading page does not require refocusing to bottom (most recent message). If the above criteria are met, testing is successful**
